docs/resources/Poll.md

```diff
@@ -0,0 +1,122 @@
+# Poll Resource
+
+A poll is... well... a poll! It holds information about a poll!
+
+![If you get it you get it](example-poll.png)
```

Comment:
```suggestion
![Example message containing a poll](example-poll.png)
```
Although accessibility may not be a top priority for developer documentation, a more helpful alt text would be nice.